dev/release/01-prepare.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+set -ue
+
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ "$#" -ne 2 ]; then
+  echo "Usage: $0 <old_version> <new_version>"
+  exit 1
+fi
+
+. $SOURCE_DIR/utils-prepare.sh
+
+old_version=$1
+new_version=$2
+version_tag="apache-arrow-${new_version}"
+
+: ${PREPARE_DEFAULT:=1}
+: ${PREPARE_VERSION_PRE_TAG:=${PREPARE_DEFAULT}}
+: ${PREPARE_TAG:=${PREPARE_DEFAULT}}
+
+
+if [ ${PREPARE_TAG} -gt 0 ]; then
+  if [ $(git tag -l "${version_tag}") ]; then
+    echo "Delete existing git tag $version_tag"
+    git tag -d "${version_tag}"
```

Comment:
Do we really need the tags? If so why do we bother deleting the old one?

Reply:
I thought it might be good to have a tag in case we want to deploy more versions of the cookbooks in the future and not only stable and dev, also to keep some history.
We only delete the new tag if it already exist, not the old one, this was only added as a guard in case we run the script twice but it could be removed if you think is not necessary.

Reply:
Ah, I misread. Ok, sounds good to me.